sled-agent/src/smf_helper.rs

```diff
@@ -17,53 +17,29 @@ pub enum Error {
 pub trait Service {
     fn service_name(&self) -> String;
     fn smf_name(&self) -> String;
-    fn should_import(&self) -> bool;
 }
 
 pub struct SmfHelper<'t> {
     running_zone: &'t RunningZone,
-    service_name: String,
+    _service_name: String,
```

Comment:
If this isn't necessary anymore (looks like we were only using it for import) can we remove it?